marshmallow/marshalling.py

```diff
@@ -203,24 +203,25 @@ def deserialize(
             will be ignored.
         :param unknown: Whether to exclude, include, or raise an error for unknown
             fields in the data. Use `EXCLUDE`, `INCLUDE` or `RAISE`.
-        :param type dict_class: Dictionary class used to construct the output.
+        :param type dict_class: Dictionary class used to construct the output when many=False.
```

Comment:
The class is used also when many=True, to construct the output of each element. I'd remove this change.